sample/Kmd.Logic.CitizenDocuments.Client.Sample/Program.cs

```diff
@@ -47,56 +46,59 @@ private static void InitLogger()
                 .CreateLogger();
         }
 
-        private static async Task Run(AppConfiguration configuration)
+        private static async Task<string> Run(AppConfiguration configuration)
         {
             var validator = new ConfigurationValidator(configuration);
             if (!validator.Validate())
             {
-                return;
+                return "The validation of provider configuration details failed";
             }
 
             using (var httpClient = new HttpClient())
             {
                 LogicTokenProviderOptions tokenProviderOptions = new LogicTokenProviderOptions()
                 {
-                    AuthorizationScope = configuration.TokenProvider.AuthorizationScope, // "https://logicidentityprod.onmicrosoft.com/ac1d197a-0e7c-4add-83c5-e1b30a08efd6/.default",
-                    AuthorizationTokenIssuer = new Uri(configuration.TokenProvider.AuthorizationTokenIssuer.ToString()), //new Uri("https://login.microsoftonline.com/logicidentityprod.onmicrosoft.com/oauth2/v2.0/token"),
-                    ClientId = configuration.TokenProvider.ClientId, //"085d3847-1b71-4203-aa52-bbb98d5ce57c",
-                    ClientSecret = configuration.TokenProvider.ClientSecret // "Ox30ERiezE+gbat7k9jtCmnfKGISFoA8AVjnJo8IgH8="
+                    AuthorizationScope = configuration.TokenProvider.AuthorizationScope,
+                    AuthorizationTokenIssuer = new Uri(configuration.TokenProvider.AuthorizationTokenIssuer.ToString()),
+                    ClientId = configuration.TokenProvider.ClientId,
+                    ClientSecret = configuration.TokenProvider.ClientSecret,
                 };
                 configuration.Citizen.SubscriptionId = configuration.SubscriptionId;
                 configuration.Citizen.Serviceuri = configuration.Serviceuri;
                 var tokenProviderFactory = new LogicTokenProviderFactory(tokenProviderOptions);
                 var citizenDocumentClient = new CitizenDocumentsClient(httpClient, tokenProviderFactory, configuration.Citizen);
-                var uploadDocument = await citizenDocumentClient.UploadAttachmentWithHttpMessagesAsync(configuration.ConfiguartionId, configuration.RetentionPeriodInDays, configuration.Cpr, configuration.DocumentType, configuration.Document, configuration.DocumentName).ConfigureAwait(false);
+                var uploadDocument = await citizenDocumentClient.UploadAttachmentWithHttpMessagesAsync(configuration.ConfigurationId, configuration.RetentionPeriodInDays, configuration.Cpr, configuration.DocumentType, configuration.Document, configuration.DocumentName).ConfigureAwait(false);
 
                 if (uploadDocument == null)
```

Comment:
Why do you explicitly handle `null` here? It looks like there's no reason to handle `null` anymore?

Reply:
Yeah correct now we don't need to check. Removed it.